lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
We need to get to this model of the world:
1) Text strings are either plain text or HTML.  You need to know which you have.  
2) Functions should be clear about whether the text they accept is plain text or HTML, and they need to be clear about what they produce.
3) Where a template is inserting a plain text string into HTML, it needs to do the escaping.
4) Escaping shouldn't happen until the text is being used in a context that needs escaping.
5) All translated text is plain text.

We aren't at this model yet, partly because some of our translated text has HTML tags in it.  But we should move toward it.